packages/codemirror-blocks/src/ui/BlockEditor.tsx

```diff
@@ -167,30 +166,6 @@ const BlockEditor = ({ options = {}, ...props }: BlockEditorProps) => {
     props.onMount(editor, buildAPI(editor, dispatch, language), passedAST);
   };
 
-  /**
-   * When the CM instance receives focus...
-   * If we have a CM cursor, let CM handle it (no-op)
-   * Otherwise grab the focusId, compute NId, and activate
-   */
-  const handleTopLevelFocus = (editor: ReadonlyCMBEditor) => {
-    setAfterDOMUpdate(() => {
-      dispatch((_, getState) => {
-        const ast = selectors.getAST(getState());
-        const focusedNode = selectors.getFocusedNode(getState());
-        const { cur } = getState();
-        if (cur != null) {
-          return; // if we already have a cursor, bail
-        }
-        const node = focusedNode || ast.getFirstRootNode();
-        dispatch(
-          activateByNid(editor, node && node.nid, {
-            allowMove: true,
-          })
-        );
```

Comment:
This was the only place where we were doing something with the redux cursor that was different than the behavior of codemirror's cursor. Our redux version of the cursor was allowed to be `null`, but codemirror's cursor is always a something. I tried all kinds of things to get the application into a state where the cursor was actually null, and couldn't figure out how to do it. When I removed this function altogether, all the tests still passed, which makes me think this function wasn't doing anything useful to begin with.